src/application.ts

```diff
@@ -89,14 +90,17 @@ export default class Application {
   async buildOperationResponse(
     data: Resource | Resource[] | void
   ): Promise<OperationResponse> {
-    const included = flatten(await this.extractIncludedResources(data)).filter(
-      Boolean
-    );
+    const included = flatten(await this.extractIncludedResources(data))
+    const uniqueIncluded = Array.from(new Set(included
+      .map((item: Resource) => `${item.type}_${item.id}`)))
```

Comment:
@spersico Shouldn't this be `foreignKeyName`-aware?

Reply:
@joelalejandro what do you mean by foreignKeyName?
I check only for repeated resources, I assume that the id and type are unique for each resource, and that's what I use, unless there's something that I'm not seeing